src/Rootfs.jl

```diff
@@ -391,14 +391,24 @@ function choose_shards(p::Platform;
 end
 
 """
-    supported_platforms()
+    supported_platforms(;exclude::Vector=[])
 
 Return the list of supported platforms as an array of `Platform`s.  These are the platforms we
 officially support building for, if you see a mapping in `get_shard_hash()` that isn't
 represented here, it's probably because that platform is still considered "in beta".
+
+Platforms can be excluded from the list by specifying an array of platforms to `exclude`.
+i.e. `supported_platforms(exclude=[Windows(:i686),Windows(:x86_64)])`
 """
-function supported_platforms()
-    return [
+function supported_platforms(;exclude::Union{Vector{<:Platform},Function}=[])
+    exclude_platforms!(exclude::Function, platforms) = filter(!exclude, platforms)
+    function exclude_platforms!(exclude::Vector{<:Platform}, platforms)
```

Comment:
Conventionally, the modified argument goes first.
```suggestion
    function exclude_platforms!(platforms, exclude::Vector{<:Platform})
```

Reply:
Actually, I think this entire thing can just be
```suggestion
    exclude_platforms!(platforms, exclude::Vector{<:Platform}) = filter!(!in(exclude), platforms)
```